packages/react/src/components/Provider/Provider.tsx

```diff
@@ -177,7 +181,7 @@ class Provider extends React.Component<WithAsProp<ProviderProps>> {
         target={target}
         {...{ rehydrate: false }}
       >
-        <ThemeProvider theme={outgoingContext}>
+        <ThemeProvider theme={outgoingContext} overwrite>
```

Comment:
`ThemeProvider` comes from Fela, we should always use **overwrite** as Fela has own shallow merging.

https://github.com/robinweser/fela/blob/v10.6.1/packages/fela-bindings/src/ThemeProviderFactory.js